### DISCUSSION TITLE
What features are enabled by `enable_subscription: true`

### DISCUSSION THREAD

**Discussion by binaryben** [mdouchement/standardfile#101] [Q&A] [answered]
I naively expected folders to be activated by this but it seems to not be. Is the `enable_subscription` item the equivalent of [doing what's outlined in this official doc](https://standardnotes.com/help/self-hosting/subscriptions)?

I'm getting "A plus or pro plan is required" messages all over the interface and can't find any definitive answers on what are server side vs client side premium features.

Sorry, I know it's out of scope for this project specifically but I'm not quite sure where the line of which part is responsible for what is quite yet.

**Comment by mdouchement** [comment, marked as answer]
It enables paid features in the client (there is nothing like that in the server side) but it seems to have issues https://github.com/mdouchement/standardfile/issues/94.
The implementation itself relies on a `sleep` in the code to work with the official clients..

**Comment by binaryben** [comment]
Okay, I will close this in favour of #94 then if it's a bug and folders should be working.
